--- gazebo_ros_control/CMakeLists.txt	2020-11-07 15:04:40.370574513 +0300
+++ gazebo_ros_control/CMakeLists.txt	2020-11-07 15:04:31.991575055 +0300
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.5.1)
 project(gazebo_ros_control)
 
+list(APPEND CMAKE_MODULE_PATH /usr/lib64/OGRE/cmake)
+
 # Load catkin and all dependencies required for this package
 find_package(catkin REQUIRED COMPONENTS
   gazebo_dev
