--- a/CMakeLists.txt	2022-09-20 14:00:22.337954696 +0300
+++ b/CMakeLists.txt	2022-09-20 14:00:05.954955754 +0300
@@ -72,6 +72,7 @@ endif()
 ########################################
 # Python bindings
 include(IgnPython)
+find_package(Python3 COMPONENTS Interpreter Development)
 find_package(PythonLibs QUIET)
 if (NOT PYTHONLIBS_FOUND)
   IGN_BUILD_WARNING("Python is missing: Python interfaces are disabled.")
