--- a/src/password-store.sh	2021-06-11 19:49:06.000000000 +0300
+++ b/src/password-store.sh	2021-06-11 19:49:06.000000000 +0300
@@ -558,7 +558,7 @@
 	elif [[ $qrcode -eq 1 ]]; then
 		qrcode "$pass" "$path"
 	else
-		printf "\e[1mThe generated password for \e[4m%s\e[24m is:\e[0m\n\e[1m\e[93m%s\e[0m\n" "$path" "$pass"
+		printf "\e[1mThe generated password for \e[4m%s\e[24m is:\n%s\n" "$path" "$pass"
 	fi
 }
 
